R/DataProcessing.R

```diff
@@ -215,7 +216,7 @@ readProjectData <- function(fileLines, progress = FALSE, qfeatures = NULL)
     }
   )
   ######debugging
- 
+  if(!is.null(qfeatures)){
```

Comment:
Please mention in a comment that qfeatures==NULL if the function was called from readClusterDataFromProjectFile().
Do we need an else for the branch to set metaboliteProfile$Annotation from the project file instead ?